package.json

```diff
@@ -31,10 +31,6 @@
     "@openapi-contrib/openapi-schema-to-json-schema": "^3.0",
     "@stoplight/json": "^3.4.0",
     "@stoplight/types": "^11.4.0",
```

Comment:
@XVincentX what about our types?

Reply:
@karol-maciaszek What do you mean?

Reply:
I mean what about moving `"@stoplight/types": "^11.4.0",` to `devDependencies`.

Reply:
Done!